content/draft/general/keyboard_shortcuts.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Keyboard shortcuts spec"
+description: ""
+date: 2020-12-01T08:48:57+00:00
+lastmod: 2020-12-01T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "general"
+weight: 100
+toc: true
+---
+
+# Keyboard Shortcuts spec
+
+## Motivation
+
+Users often use keyboard shortcuts to more effectively navigate applications. This spec aims to document the keyboard shortcuts available as well as any preconditions.
+
+## Designs
+
+* [designs](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=355%3A82493)
+
+## Functional Requirements
+
+### Key definition
+
+The requirements will use the keys `CMD` and `OPT` which are defined for each platform as follows:
+
+- `CMD`
+    - macOS: `⌘` or `CMD`
+    - Windows: `Ctrl`
+    - Linux: `Ctrl`
+- `OPT`
+    - macOS: `⌥` or `OPT`
+    - Windows: `Alt`
+    - Linux: `Alt`
+
+### Keyboard shortcuts help
+
+- `CMD + /` MUST open a modal listing available key shortcuts
+- The Modal listing available keyshortcuts MUST display the correct keys depending on the platform, for example `CMD + C` on macOS and `Ctrl + C` on Windows.
```

Comment:
Might want to specify that this an application-wide, global key command that applies everywhere in the app (if that is indeed the case. If not, specify as well)

Reply:
Good point! Yes this should be global app-wide, the user should be able to invoke the keyboards shortcuts overlay irrespective of where they are in the app, and even if a modal dialogue is open.  This keyboards shortcuts overlay should always appear on top of whatever else is on the screen.